nurseconnect/views.py

```diff
@@ -1,8 +1,14 @@
+from django.contrib.auth import authenticate, login
+from django.contrib.auth.models import User
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect
 from django.shortcuts import render
 from django.utils.translation import get_language_from_request
```

Comment:
An open line is needed here to separate django imports from module-level imports